modules/openvino_code/shared/model.ts

```diff
@@ -3,19 +3,24 @@ import { Features } from './features';
 enum ModelId {
   CODE_T5_220M = 'Salesforce/codet5p-220m-py',
   DECICODER_1B_OPENVINO_INT8 = 'chgk13/decicoder-1b-openvino-int8',
+  STABLECODE_COMPLETION_ALPHA_3B_4K_OPENVINO_INT8 = 'chgk13/stablecode-completion-alpha-3b-4k-openvino-int8',
 }
 
 export enum ModelName {
   CODE_T5_220M = 'codet5p-220m-py',
   DECICODER_1B_OPENVINO_INT8 = 'decicoder-1b-openvino-int8',
+  STABLECODE_COMPLETION_ALPHA_3B_4K_OPENVINO_INT8 = 'stablecode-completion-3b',
```

Comment:
New model name should also be added to side panel select options: `side-panel-ui/src/components/sections/ServerSection/ModelSelect/ModelSelect.tsx` (`const options: SelectOptionProps<ModelName>[]`, L6-L9)

Reply:
Also, I would suggest to make it more dynamic, e.g.
```typescript
const options: SelectOptionProps<ModelName>[] = Object.values(ModelName).map((value) => ({ value }));
```